templates/registration/public_profile.html

```diff
@@ -9,23 +9,27 @@
 {% block content %}
     <div class="center">
         <br/><img class="public-profile-crop" src="{{ profile_user.userprofile.display_avatar_url }}" alt="Profile" />
-        <h3 class="profile-padding">{{ profile_user.userprofile.get_display_name }}'s<br/> Profile</h3>
+        <h3 class="public-profile-name">{{ profile_user.userprofile.get_display_name }}</h3>
     </div>
+
+    <div>
+        <h2 class="green-color">{{ percentage|floatformat:2|intcomma }}%</h2>
+        <h4 class="green-color"><i class="fa fa-arrow-up" aria-hidden="true"></i>${{ total_earned|floatformat:2|intcomma }} | <span style="color: #4d96ad">${{ total_investments|floatformat:2|intcomma }}</span></h4>
```

Comment:
The up arrow will show even when the user's total earned is 0. Is that what you want?